src/config.mts

```diff
@@ -0,0 +1,41 @@
+import path from 'path';
+import { pathToFileURL } from 'url';
+
+let configurationInstance: Config | undefined = undefined;
```

Comment:
Do we really need this global state here? Why not just keep configuration instance at `cli.mts`?

Reply:
Fixed by 76cdee7b0ad48d03e2bda165ac16b49b23ff9e07.